components/kyma-environment-broker/internal/broker/poller.go

```diff
@@ -0,0 +1,47 @@
+package broker
+
+import (
+	"time"
+
+	"github.com/pkg/errors"
```

Comment:
same here, please avoid usage of this deprecated package https://github.com/kyma-project/control-plane/pull/2234